packages/frontend/components/composed/article/ArticleContributor/ArticleContributor.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { graphql } from "react-relay";
+import { useMaybeFragment } from "@wdp/lib/api/hooks";
+import * as Styled from "./ArticleContributor.styles";
+import ContributorAvatar from "components/composed/contributor/ContributorAvatar";
+import ContributorName from "components/composed/contributor/ContributorName";
+import { ArticleContributorFragment$key } from "@/relay/ArticleContributorFragment.graphql";
+
+export default function ArticleContributor({ data }: Props) {
+  const contributions = useMaybeFragment(fragment, data);
+  const authorContributions = contributions?.edges.filter(
+    (edge) => edge.node.role?.toLowerCase() === "author"
+  );
+  const contributionToShow = authorContributions?.length
+    ? authorContributions[0].node
+    : null;
+  const contributor = contributionToShow?.contributor;
+
+  return contributor &&
+    (contributor.__typename === "PersonContributor" ||
+      contributor.__typename === "OrganizationContributor") ? (
+    <section className="a-bg-custom10">
+      <div className="l-container-wide">
+        <Styled.Inner>
+          <Styled.AvatarWrapper>
```

Comment:
Might be good to check with Lael if an empty avatar should show here. In other areas of the app we're hiding the avatar if it doesn't exist (unless it's a group of contributors and one or more have an image). The easiest way to check is on the image's storage property. 👍